SplunkRumWorkspace/SmokeTest/SmokeTest/ViewController.swift

```diff
@@ -18,12 +18,23 @@ limitations under the License.
 import UIKit
 import WebKit
 import SplunkRum
+import CoreLocation
 
-class ViewController: UIViewController, WKUIDelegate {
-
+class ViewController: UIViewController, WKUIDelegate, CLLocationManagerDelegate {
+    @IBOutlet var lblLat: UILabel!
+    @IBOutlet var lblLongi: UILabel!
+    @IBOutlet var lblAdd: UILabel!
+    var locationManager: CLLocationManager!
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
+        locationManager = CLLocationManager()
+        locationManager.delegate = self
+        locationManager.desiredAccuracy = kCLLocationAccuracyBest
+        locationManager.requestAlwaysAuthorization()
```

Comment:
This will prompt the end user, won't it?

Reply:
no, it's not because we did not add permission in Splunk RUM plist file.